nncf/onnx/statistics/aggregator.py

```diff
@@ -34,6 +34,7 @@ class ONNXStatisticsAggregator(StatisticsAggregator):
     def collect_statistics(self, model: onnx.ModelProto) -> None:
         self._nncf_graph = NNCFGraphFactory.create(model)
         self._onnx_graph = ONNXGraph(model)
+        self._registered_weights = set()
```

Comment:
Could you avoid adding a new attribute?

Reply:
Statistic_point contains several statistic collectors, so we need to check all collectors have registered constant at least ones. I suggest to keep it as is